Z/ZipFlow/build_tarballs.jl

```diff
@@ -0,0 +1,23 @@
+using BinaryBuilder
+
+name = "ZipFlow"
+version = v"1.0.0"
+
+sources = [GitSource("https://github.com/madler/zipflow.git",
+                     "d4d73304252504bbade9aa6f34332bbb00de6664")]
+
+script = raw"""
+    cd ${WORKSPACE}/srcdir/zipflow/
+    cc zipflow.c -shared -std=c99 -o libzipflow.${dlext} -I${includedir} -L${libdir} -lz
+    mv libzipflow.${dlext} ${libdir}/
+    install_license LICENSE
+    """
+
+platforms = supported_platforms(; experimental=true)
```

Comment:
```suggestion
platforms = supported_platforms()
```

Reply:
Is the experimental flag no longer a thing?

Reply:
It does nothing.